### ISSUE TITLE
What is cell balancing and how does it work on a Audi e-tron

### ISSUE THREAD

**Issue by etrondriver** [electrichasgoneaudi/etron-issues#86]
#### Description from Audi Self Service program 675

![bilde](https://user-images.githubusercontent.com/59776765/92300952-9a6d3000-ef5f-11ea-9c2d-a3fa8e569b2a.png)

In this example, a cell is 100% charged and the charging procedure is complete. However, the high-voltage battery charge level is
only 92.5 %. Balancing means that this cell is now discharged via a resistor and can thereby continue to be charged until all cells have reached the same charge level. This allows the high-voltage battery to achieve its maximum capacitance.
To do this, the battery regulation [control unit J840](https://github.com/etrondriver/etron-issues/issues/87) compares the voltages of the cell groups. If cell groups have a high cell voltage, the battery modules control unit responsible receives the balancing information. Balancing is performed when voltage differences of greater than approx. 1% occur when the high-voltage battery is being charged. After the ignition has been switched off, the battery regulation [control unit J840](https://github.com/etrondriver/etron-issues/issues/87) checks whether balancing is necessary and triggers it if necessary. Only the control units on the sub CAN are active when this is done. Balancing is performed at charge levels greater than 30%.

It is not possible to check the cell balance without extra tools. [ODBEleven](https://obdeleven.com/en/) is one of these tools that can be used. The below screenshot show that one cell set has 10% SOC and another has 13% soc.  Audi e-tron has 108 cell sets with 3 or 4 cells in parallel depending on version.

![bilde](https://user-images.githubusercontent.com/59776765/112756113-e3f89780-8fe3-11eb-9ec3-796bd48e211e.png)

If the battery is not properly balanced it would affect the range. 100% charged would not be normal 100%. It seems like when you run soc low there is a higher risk that the battery would be out of balance. 

In the below diagram you see how an e-tron 55 with the original 83.6kWh battery lost capacity because of running the car to a low level and the battery cells got out of balance. For a long time, only 75kWh was available. Keeping it parked on a little above 30% for some days got the original capacity back

![bilde](https://user-images.githubusercontent.com/59776765/112714374-554c2380-8eda-11eb-9f7f-ebf3de05c024.png)
